packages/react/src/providers/QuilttAuthProvider.tsx

```diff
@@ -26,6 +22,15 @@ export const GraphQLClient = new QuilttClient({
 export const QuilttAuthProvider: FC<QuilttAuthProviderProps> = ({ token, children }) => {
   const { session, importSession } = useQuilttSession()
 
+  // @todo: extract into a provider so it can accessed by child components
+  const GraphQLClient = useMemo(
```

Comment:
May make sense to rename to lowerCase

Reply:
Yeah, if it's not a global const anymore